Patient Registry System/PatientRegistrySystem.Services/IUserService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using PatientRegistrySystem.DB.Entities;
+using PatientRegistrySystem.Domain.Dto;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace PatientRegistrySystem.Services
+{
+    public interface IUserService
+    {
+        Task<IEnumerable<UserDto>> GetAllUsersAsync();
+        Task<UserDto> GetUserAsync(int id);
+        Task<bool> UpdateUserAsync(int userId, UserDto updatedUser);
```

Comment:
the userid should be filled in the updatedUser, no need to pass it alone to the function